client/src/pages/Plan/Year.js

```diff
@@ -165,8 +165,7 @@ const Year = ({ year, data, startYear, updateData, courses, setStartYear }) => {
                                 width="70%"
                                 size="lg"
                                 value={editStartYear + year}
-                                // TODO (Dalton): Add editing functionality to years
-                                // onChange={(e) => new RegExp("^[0-9]+$").test(e) && setEditStartYear(e - year)}
+                                onChange={(e) => new RegExp("^[0-9]+$").test(e.target.value) && setEditStartYear(e.target.value - year)}
```

Comment:
Can we constrain this to having a year length of 4? Very doubtful that it will ever be 5 in the lifetime of this application

Reply:
done